### ISSUE TITLE
Mobile navigation icon overlaps text; how to force tab titles into navigation menu?

### ISSUE THREAD

**Issue by jpskycak** [academicpages/academicpages.github.io#205]
When I view on mobile, the black navigation icon overlaps the text of the "Teaching" tab that is displayed to the left. Is there any way to force the "Teaching" tab to be displayed in the navigation icon, rather than outside of it where it overlaps?
![52586331_614427945667258_3709005655131029504_n](https://user-images.githubusercontent.com/13306854/53292000-db806380-3770-11e9-860a-9b3d553dffb3.png)

**Comment by davidolohowski** [comment]
I have the same issue as yours. I added another page about my research areas but it does not go into the tab when viewed on mobiles. No idea what to do. Have you figured it our yet?

**Comment by jpskycak** [comment]
Hi @davidolohowski, I haven't figured it out. But it hasn't been a functional inconvenience to me, moreso just a design issue.

**Comment by davidolohowski** [comment]
@jpskycak I guess it's fine. It's just a bit annoying viewing it on phone. Thanks anyway.

**Comment by davidolohowski** [comment]
@jpskycak I think there might be some bugs. I switched the place of the link and I put research behind publications and now things worked. You can try switch tutoring and teaching behinid books, it might work.

**Comment by jpskycak** [comment]
@davidolohowski yeah, I see that too. Switching around the labels results in different positions due to different numbers of characters, and if you can make it so that the rightmost label is far enough from the box not to overlap but close enough that it doesn't try to squeeze in another label, then it works.

Haven't tried this myself, but based on that finding, maybe inserting some spaces into a tab name could allow you to force a particular tab to move into the box. The resulting spacing would look weird on desktop layout though, so probably not worth it.
